globus_portal_framework/templates/globus-portal-framework/v3/components/search-nav.html

```diff
@@ -0,0 +1,14 @@
+{% load static %}
+
+<div class="subnav mt-auto bottom-border-0">
+  <ul class="nav nav-tabs nav-search-nav justify-content-center">
+    <li class="mt-auto">
+      <a class="btn btn-primary btn-lg border-0 squared {% if active == 'search' %}active{% endif %}" role="button"
+        href="{% url 'search' globus_portal_framework.index %}">Search</a>
+    </li>
+    <li class="mt-auto">
+      <a class="btn btn-primary btn-lg border-0 squared {% if active == 'about' %}active{% endif %}" role="button"
```

Comment:
You should be able to do this with the `is_active` template tag in DGPF. And then you won't need to pass context and manually check this. There should be examples in the other templates, it should look something like this: 

```
<li class="breadcrumb-item {% is_active request 'search-about' index=globus_portal_framework.index %}">
```